recipe/bld.bat

```diff
@@ -0,0 +1,6 @@
+
+nmake /f makefile.vc
+if errorlevel 1 exit 1
+
```

Comment:
`ctest` is part of `cmake` but this isn't a cmake build and there is no `test` command in Windows unless you are running inside a .bat script. There is also no `make check` type target for freexl. I've adapted `run_test.py` from the kealib feedstock to check the libraries load and left the check for the static lib on Unix.